docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+## UID2 Operator Error Codes
+
+The following table lists possible errors while setting up a Private Operator.
+
+:::note
+This applies only to Private Operators starting from Q2 2024.
```

Comment:
This is not very clear - it somewhat sounds like existing private operator will magically show these error code from Q2 2024. Should be more like:

Only Private Operators versions released on and after Q2 2024 will present these error codes if private operator startup issues are encountered

Reply:
change to 

"Error codes for private operator startup issues will only appear in versions released from Q2 2024 onward." 

?

Reply:
Yes that's better. Though, best not to use future tense. My suggestion:
Error codes for Private Operator startup issues apply to versions released from Q2 2024 and later..